packages/@uppy/companion-client/types/index.d.ts

```diff
@@ -10,20 +10,31 @@ export interface TokenStorage {
   removeItem: (key: string) => Promise<void>
 }
 
+type CompanionHeaders = Record<string, unknown>
```

Comment:
What is unknown about the headers? A header can't be anything other than a string (perhaps an array)?
```suggestion
type CompanionHeaders = Record<string, string>
```